src/world/area_gv/gv_01/gv_01_1_main.c

```diff
@@ -44,6 +44,113 @@ EvtScript N(EVS_DropLastLetter) = {
     //@bug not terminated!
 };
 
+#if VERSION_PAL
+EvtScript N(EVS_E8C6F8) = {
+    EVT_CALL(EnableGroup, 20, 0)
```

Comment:
The highest `MODEL_` constant defined for `gv_01` is 0x13 (19).

Reply:
mapfs `gv_01_shape.bin` must have changed between releases.. we should get headers in for the other versions in ASAP so we don't introduce problems where we reference the wrong model/collider/group/zone ids.

@marijnvdwerf Can you throw a `// TODO` on this line?

@HailSanta think we can get Star Rod to split JP/PAL/IQUE maps? Or think its better to wait until we have actual mapfs shape/hit support in decomp & then properly generate all the headers?

Reply:
We can wait to do them properly, TODO this for now. This is the GAME OVER map where the letters are models in the map, so it makes sense they'd edit it to accommodate other languages.